ui/src/app/styles/cards.css

```diff
@@ -121,24 +121,30 @@
 }
 
 .help-tip {
-  height: 128px;
-  width: 293px;
+  height: 3.25rem;
```

Comment:
I would use a margin top/bottom rather than a fixed height here, I noticed this issue locally when reducing the window size: 
![image](https://user-images.githubusercontent.com/822298/43854835-dc983758-9af8-11e8-9d8b-bb86635b979b.png)